go.mod

```diff
@@ -14,5 +15,5 @@ require (
 	github.com/spf13/cobra v1.1.1
 	github.com/stretchr/testify v1.7.0
 	golang.org/x/mod v0.4.1
-	k8s.io/apimachinery v0.20.2
+	k8s.io/apimachinery v0.18.6
```

Comment:
Note to reviewers: this was necessary in order to avoid this issue: https://github.com/kubernetes/client-go/issues/741